docs/how-tos/hello-world-module.md

```diff
@@ -406,7 +694,23 @@ With the implementation written, it's time to test your module locally:
    This environment is where the local module will run.
    `viam-server` does not need to run inside this environment.
 
-1. Make sure your machine's instance of `viam-server` is live and connected to the [Viam app](https://app.viam.com).
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+1. Use Go to compile your module into a single executable:
+
+   - Navigate to your <file>hello-world</file> module directory in your terminal.
+   - Run `make` and then `make setup` to install dependencies.
+   - Run `go build` to compile your entry point (main program) file <file>main.go</file> and all other <file>.go</file> files in the directory, building your module and all dependencies into a single executable file.
```

Comment:
We can simplify this and make it copy pastable. I don't want to have to read through text to skim for commands I need to run.
```suggestion
1. From within the <file>hello-world</file> directory, compile your module into a single executable:

   ```sh {class="command-line" data-prompt="$" data-output="5-10"}
   make
   make setup
   go build
   ```
```

Reply:
Shouldn't we run 
```
make setup
make
```
It looks like the make command already runs go build?

Reply:
I only got build errors when I ran go build, not make or make setup, so I used that to debug. Will ask SME.

Reply:
```suggestion
1. From within the <file>hello-world</file> directory, compile your module into a single executable:
	
	   ```sh {class="command-line" data-prompt="$" data-output="5-10"}
	   make setup
	   make build
```